projects/firmware_102/src/main.c

```diff
@@ -0,0 +1,44 @@
+#include <stdbool.h>
```

Comment:
It's good that you put comments in your code and I think you did this for learning and to remind yourself of what everything does in the library. I encourage you to continue doing this if you are not sure what something does. However, when you are actually writing code and pushing it into the repo some of these comments are sort of self-explanatory. For example, I don't think you need to make a commend that the wait.h header includes the wait function.